src/java.base/share/classes/java/util/zip/ZipEntry.java

```diff
@@ -37,6 +37,11 @@
 
 /**
  * This class is used to represent a ZIP file entry.
+ * <P>
+ * The combined length of the entry name, the extra field data, the
```

Comment:
"bytes" is the last word here, so technically this is correct. But an external reader may easily think this refers to the string lengths, not the byte encoded lengths.

So  maybe being slightly more explicit that the encoded length depends on the chosen charset would reduce ambiguity?

Reply:
I am not sure it is needed, but could tweak to 

> The combined length, after encoding, of the ......

thoughts?

Reply:
> > The combined length, after encoding, of the ......
> 
> thoughts?

I like it. Disambiguating but also succinct.

Reply:
Set to be updated as part of the next PR push